### PR TITLE
plots: smooth: example

### LINKED ISSUE #1439: plots: add smooth template example
https://github.com/iterative/dvc/pull/4046#issuecomment-644437731

CC @pared @dmpetrov

### PR DESCRIPTION
**Pull request by pared**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏
Fixes #1439

### CODE REVIEW

File: content/docs/command-reference/plots/diff.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ To compare uncommitted changes of a metrics file and its last committed version:
 
 ```dvc
 $ dvc plots diff --targets logs.csv --x-label x
-file:///Users/dmitry/src/plots/logs.html
+file:///Users/usr/src/plots/logs.html
```
Comment (pared) [inline content/docs/command-reference/plots/diff.md:110]: found this one, during reading of plot docs, I don't think it needs separate PR

Comment (efiop) [reply content/docs/command-reference/plots/diff.md:110]: What about it though?

File: content/docs/command-reference/plots/index.md [hunk 2]
```diff
@@ -99,6 +99,12 @@ option of `dvc plots show` and `dvc plots diff`. For templates in the
 `.dvc/plots/` directory, the path and the json extension are not required: you
 can specify only the base name e.g. `--template scatter`.
 
+DVC has built-in following plot types:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/index.md:102]:
```suggestion
DVC has the following built-in plot templates:

```

File: content/docs/command-reference/plots/index.md [hunk 3]
```diff
@@ -198,6 +204,27 @@ file:///Users/usr/src/plots/logs.html
 
 ![](/img/plots_show_field.svg)
 
+## Example: Smooth plot
+
+In some cases we would like to smooth our plot. In this example we will use plot
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/index.md:209]:
```suggestion
In some cases we would like to smooth our plot. In this example we will use a plot
```

File: content/docs/command-reference/plots/index.md [hunk 4]
```diff
@@ -198,6 +204,27 @@ file:///Users/usr/src/plots/logs.html
...
+with 1000 data points:
+
+```dvc
+$ dvc plots show data.csv
+file:///Users/usr/src/plots/plots.html
+```
+
+![](/img/plots_show_no_smooth.svg)
+
+We can use `-t` option and `smooth` template to make it less noisy:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/index.md:219]:
```suggestion
We can use the `-t` option and `smooth` template to make it less noisy:
```

**Comment by pared** [comment]
@jorgeorpinel squashed and reformatted

**Pull request merged**